public/app/features/canvas/runtime/element.tsx

```diff
@@ -191,8 +193,13 @@ export class ElementState implements LayerElement {
         this.div.style[key as any] = (this.sizeStyle as any)[key];
       }
 
-      for (const key in this.dataStyle) {
-        this.div.style[key as any] = (this.dataStyle as any)[key];
+      const elementType = this.options.type;
+      // SVG elements have their own styles
+      // TODO: This is a hack, we should have a better way to handle this
```

Comment:
This will lead to some glitches / bugs, such as when changing element types from rectangle to one of the svg elements

We should create a follow-up github issue to address this bug ideally as part bug bash / feature hardening phase

https://github.com/grafana/grafana/assets/22381771/c779e3ed-131a-4fdc-a865-26272c0db51b

Reply:
https://github.com/grafana/grafana/issues/84869